src/App.module.scss

```diff
@@ -1,38 +1,25 @@
-.App {
-  text-align: center;
-}
-
-.appLogo {
-  height: 40vmin;
-  pointer-events: none;
+.navbar ul {
```

Comment:
let's try to only use multiple selectors where its meaningful.
Nesting/multiple selectors in other situations tends to just make life a bit trickier due to needing to think about CSS selection rules when maintaining the code.

In this case, I think it would be more maintainable if we just gave this `ul` a CSS class.